src/command_modules/azure-cli-interactive/azclishell/telemetry.py

```diff
@@ -52,9 +54,8 @@ def conclude(self):
         self.end_time = str(datetime.datetime.now())
         self.track_event('Run', {'start time': self.start_time,
                                  'end time': self.end_time})
```

Comment:
Following this sample for start and end time:
https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/telemetry.py#L150